matchbox/resource_group.go

```diff
@@ -53,11 +59,23 @@ func resourceGroupCreate(d *schema.ResourceData, meta interface{}) error {
 		selectors[k] = v.(string)
 	}
 
+	metadata := map[string]interface{}{}
+	if j, ok := d.GetOk("metadata_json"); ok {
+		err := json.Unmarshal([]byte(j.(string)), &metadata)
+		if err != nil {
+			return err
+		}
+	}
+
+	for k, v := range d.Get("metadata").(map[string]interface{}) {
+		metadata[k] = v.(string)
```

Comment:
Can you please add to the TestResourceGroup metadata to include a non-string value (such as a number) to prove this is not breaking.